karbon/v1alpha1/karbon_registration_api.go

```diff
@@ -0,0 +1,71 @@
+package v1alpha1
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/nutanix-cloud-native/prism-go-client/internal"
+)
+
+type V1Aplha1 struct {
```

Comment:
@deepakm-ntnx Should `V1Alpha1` be an interface instead?
e.g. 
```go
type V1Alpha1 interface {
    KubernetesRegistrationService
    MetaService
    ...
}
```
The whole interface `KubernetesRegistrationService` can be moved to a similar `V1Beta1` whenever the APIs evolve.

Reply:
@thunderboltsid I think it may be better to keep the v1alpha1 as a struct for now and we could change when the v1beta1 need to be added.